util/src/kvdb.rs

```diff
@@ -123,16 +126,65 @@ pub struct CompactionProfile {
 impl Default for CompactionProfile {
 	/// Default profile suitable for most storage
 	fn default() -> CompactionProfile {
+		CompactionProfile::ssd()
+	}
+}
+
+/// Given output of df command return Linux rotational flag file path.
+pub fn rotational_from_df_output(df_out: Vec<u8>) -> Option<PathBuf> {
```

Comment:
right, but it seems a bit cleaner to have all linux-specific logic (even stuff like checking the flag file) within functions which are only compiled on linux to avoid any accidental leakage. we can have stubs for unsupported platforms that will return SSD.